packages/@react-spectrum/color/test/ColorWheel.test.tsx

```diff
@@ -341,7 +341,7 @@ describe('ColorWheel', () => {
       let defaultColor = parseColor('hsl(0, 100%, 50%)');
       let {container: _container, getByRole} = render(<ControlledHSL defaultValue={defaultColor} onChange={onChangeSpy} onChangeEnd={onChangeEndSpy} />);
       let slider = getByRole('slider');
-      let container = _container?.firstChild?.firstChild?.firstChild as HTMLElement;
+      let container = _container?.firstChild?.firstChild?.firstChild?.firstChild as HTMLElement;
```

Comment:
😂

Reply:
yeah... we have far too many of these, they are very silly